src/ECharts/Commands.purs

```diff
@@ -233,6 +251,9 @@ missingValue = set "" undefinedValue
 valuePair ∷ ∀ i. String → Number → DSL (value ∷ I|i)
 valuePair a b = set "value" $ toForeign [toForeign a, toForeign b]
 
+titles ∷ ∀ i. DSL TP.TitlesI → DSL (title ∷ I|i)
+titles = set "title" <<< buildArr

```

Comment:
Actually, the option `title` in Echarts 3.0 accepts both a `title` object and a array of 'title' objects, as following:

```javascript
title: {
          text: 'T'
       }
```
or
```javascript
title: [{
          text: 'T1'
       },
       {
          text: 'T2'
       }]
```